tests/lib/prepare.sh

```diff
@@ -86,6 +86,9 @@ EOF
         echo "/etc/systemd/system/snapd.service.d/local.conf vanished!"
         exit 1
     fi
+    if [ "${APT_AUTOCLEAN:-}" = "1" ]; then
```

Comment:
Lets not name this APT_AUTOCLEAN. APT is a term specific for Ubuntu/Debian. PACKAGE_AUTOCLEAN would be a better fit here.